index.js

```diff
@@ -5,8 +5,12 @@ const {source: repository} = /(?<repository>[\w.-]{1,100})/;
 const {source: reservedRepository} = /(?<!\/\.{1,2})/;
 const {source: issueNumber} = /(?<issueNumber>[1-9]\d{0,9})/;
 const {source: initialDelimiter} = /(?<!\w)/;
-const fullRegex = `${initialDelimiter}(?:${organization}(?:\\/${repository})?)?${reservedRepository}#${issueNumber}\\b`;
+const fullRegex = `${initialDelimiter}(?:(?:${organization}(?:\\/${repository})?)?${reservedRepository}#)${issueNumber}\\b`;
+
+export default function issueRegex(prefix) {
+	if (prefix) {
+		return new RegExp(fullRegex.replace('#', '#|' + prefix), 'gi');
```

Comment:
There shouldn't be a need to escape this value and I don’t think this regex could support a prefix that looks like `[GH]123` anyway. Perhaps this can be mentioned in the readme: 

> `prefix` is added unescaped to the regex, keep it simple.

Reply:
Note added to readme and types